addons/website_slides/controllers/main.py

```diff
@@ -301,17 +301,39 @@ def create_slide(self, *args, **post):
     # --------------------------------------------------
     @http.route('/slides/embed/<int:slide_id>', type='http', auth='public', website=True, sitemap=False)
     def slides_embed(self, slide_id, page="1", **kw):
+        """
+        Detect if the slide is embedded in a website outside of the Odoo database.
+        The detection is there to differentiate cases where the slide is present on a webpage that is different than
+        the website for which the slide was created
+
+        This is how we decide if the slide is embedded or not:
+        if referer_url is Falsly -> Not embedded (it's the default behavior)
```

Comment:
That seems like the wrong default. Surely if we don't know whether it's embedded it's better to have the control possibly show twice than to hacve them possibly show never?

Reply:
That's true, in the beginning we wanted to avoid embedding it since the embedded view had visual issues
But furthermore we count the number of times a slide was shown, and separate between shows of the embedded version, and shows of the "normal" version so it's up to the people at slides to decide (I'll ask them)

Reply:
It is globally the "old default" and I fear that having a default to "is embedded" will display broken options more often. As this is the first time I hear from this request and as we target stable, I would keep the current default behavior.

Reply:
> It is globally the "old default" and I fear that having a default to "is embedded" will display broken options more often. 

Are the options broken when they're displayed in both the page and the embed? Or are they just... displayed in both which is a bit ugly but that's about it?

Reply:
@xmo-odoo there was a difference between PDF view embedded or not, when embedded there is a banner at the top of the PDF containing a few buttons to Share, Email, or Embed (which are located outside of the PDF view and below of the page when **not** embedded)
And it's 2 different view (embedded one is button that create a dropdown menu, not embedded one is the screenshot
![2021-09-22_11-40](https://user-images.githubusercontent.com/32837941/134320936-2856bed5-6118-4a16-85ba-22c139e17193.png)

Reply:
Sure, but my question is whether using the embedded view in non-embedded contexts is actively problematic or just ugly. That's what's relevant to the default. If the embedded view in non-embedded contexts just leads to having the share stuff twice, then it strikes me as a better idea than having the non-embedded view in embedded contexts and no share controls.

Reply:
It is quite ugly, leading to a globally broken design. Better avoid displaying them when not necessary.